<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>795</width>
    <height>550</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>795</width>
    <height>550</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>795</width>
    <height>550</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>GAMING :)</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow{
background-color: rgb(213,213,213);
}
QPushButton{
	font-family:&quot;Lucida Console&quot;, &quot;Courier New&quot;, monospace;
}
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="inventory">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>250</y>
      <width>140</width>
      <height>140</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	
	
}</string>
    </property>
    <property name="text">
     <string>Inventory / Weapon</string>
    </property>
    <property name="iconSize">
     <size>
      <width>50</width>
      <height>50</height>
     </size>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="playerHealth">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>400</y>
      <width>111</width>
      <height>61</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::RightToLeft</enum>
    </property>
    <property name="text">
     <string>Player.health</string>
    </property>
   </widget>
   <widget class="QPushButton" name="west_button">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>360</y>
      <width>100</width>
      <height>111</height>
     </rect>
    </property>
    <property name="text">
     <string>Go West</string>
    </property>
   </widget>
   <widget class="QPushButton" name="north_button">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>250</y>
      <width>100</width>
      <height>111</height>
     </rect>
    </property>
    <property name="text">
     <string>Go North</string>
    </property>
   </widget>
   <widget class="QPushButton" name="east_button">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>250</y>
      <width>100</width>
      <height>111</height>
     </rect>
    </property>
    <property name="text">
     <string>Go East</string>
    </property>
   </widget>
   <widget class="QPushButton" name="south_button">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>360</y>
      <width>100</width>
      <height>111</height>
     </rect>
    </property>
    <property name="text">
     <string>Go South</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="enemyButton_1">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>300</y>
      <width>110</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QRadioButton" name="enemyButton_2">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>300</y>
      <width>110</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QRadioButton" name="enemyButton_3">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>350</y>
      <width>110</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QRadioButton" name="enemyButton_4">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>350</y>
      <width>110</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QRadioButton" name="enemyButton_5">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>400</y>
      <width>110</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QRadioButton" name="enemyButton_6">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>400</y>
      <width>110</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="attackButton">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>460</y>
      <width>90</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>Attack</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>19</x>
      <y>270</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
		
	font: 14pt &quot;Cambria&quot;;
}</string>
    </property>
    <property name="text">
     <string>Enemies in Room:</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="itemButton_1">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>300</y>
      <width>110</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QRadioButton" name="itemButton_2">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>300</y>
      <width>110</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QRadioButton" name="itemButton_3">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>350</y>
      <width>110</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QRadioButton" name="itemButton_4">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>350</y>
      <width>110</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QRadioButton" name="itemButton_5">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>400</y>
      <width>110</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QRadioButton" name="itemButton_6">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>400</y>
      <width>110</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="takeButton">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>460</y>
      <width>90</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>Take</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>249</x>
      <y>270</y>
      <width>140</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
		
	font: 14pt &quot;Cambria&quot;;
}</string>
    </property>
    <property name="text">
     <string>Items in Room:</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="output">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>791</width>
      <height>231</height>
     </rect>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>268</y>
      <width>20</width>
      <height>191</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="Line" name="line_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>450</y>
      <width>446</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_3">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>260</y>
      <width>446</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_4">
    <property name="geometry">
     <rect>
      <x>439</x>
      <y>267</y>
      <width>16</width>
      <height>191</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>795</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuZork">
    <property name="title">
     <string>Zork</string>
    </property>
   </widget>
   <addaction name="menuZork"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
